client/extensions/woocommerce/components/woocommerce-colophon/index.js

```diff
@@ -3,25 +3,45 @@
  * External dependencies
  */
 import React from 'react';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import ExternalLink from 'components/external-link';
```

Comment:
This should be in the "internal dependencies" section.

Reply:
Updated in: 45c819c